src/components/CallView/CallView.vue

```diff
@@ -646,9 +646,8 @@ export default {
 			if (this.isSidebar) {
 				return
 			}
-			this.$store.dispatch('startPresentation')
 			this.$store.dispatch('selectedVideoPeerId', peerId)
-			this.isLocalVideoSelected = false
+			this.$store.dispatch('startPresentation')
```

Comment:
Couldn't reproduce an issue here, but `isLocalVideoSelected` was unused in the code, and method now is aligned with `handleClickLocalVideo` by behavior :+1:

Reply:
There was no issue, just to align with `handleClickLocalVideo`